var/spack/repos/builtin/packages/openpmd-api/package.py

```diff
@@ -45,18 +46,19 @@ class OpenpmdApi(CMakePackage):
     variant("shared", default=True, description="Build a shared version of the library")
     variant("mpi", default=True, description="Enable parallel I/O")
     variant("hdf5", default=True, description="Enable HDF5 support")
-    variant("adios1", default=False, description="Enable ADIOS1 support")
+    variant("adios1", default=False, description="Enable ADIOS1 support", when="@:0.15")
     variant("adios2", default=True, description="Enable ADIOS2 support")
     variant("python", default=False, description="Enable Python bindings")
 
     depends_on("cmake@3.15.0:", type="build")
+    depends_on("cmake@3.22.0:", type="build", when="@0.16.0:")
     depends_on("catch2@2.6.1:2", type="test")
     depends_on("catch2@2.13.4:2", type="test", when="@0.14.0:")
     depends_on("catch2@2.13.10:2", type="test", when="@0.15.0:")
     depends_on("mpi@2.3:", when="+mpi")  # might become MPI 3.0+
     depends_on("nlohmann-json@3.9.1:")
     depends_on("mpark-variant@1.4.0:", when="@:0.14")  # pre C++17 releases
-    depends_on("toml11@3.7.1:3.8.1", when="@0.15.0:")
+    depends_on("toml11@3.7.1:3", when="@0.15.0:")
```

Comment:
Ah, we support 4.0+ now, too:
```suggestion
    depends_on("toml11@3.7.1:3", when="@0.15.0:")
    depends_on("toml11@3.7.1:", when="@0.16.0:")
```